examples/bri-3/src/app.module.ts

```diff
@@ -4,13 +4,17 @@ import { CommunicationModule } from './bri/communication/communication.module';
 import { IdentityModule } from './bri/identity/identity.module';
 import { TransactionModule } from './bri/transactions/transactions.module';
 import { WorkgroupModule } from './bri/workgroup/workgroup.module';
+import { ProofModule } from './bri/zkp/zkp.module';
+import { LoggingModule } from './shared/logging/logging.module';
 
 @Module({
   imports: [
     IdentityModule,
     WorkgroupModule,
     TransactionModule,
     CommunicationModule,
+    ProofModule,
```

Comment:
should this not be `zkpModule`? @biscuitdey since it will do generation and verification

Reply:
I think it should be spelled out for the same reason we have chosen clarity in all full-length naming schemes in this project: zeroKnowledeProofModule.